<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuManualSearch</class>
 <widget class="QWidget" name="MenuManualSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuManualSearch</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual Search</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_TPStart">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TP Start :</string>
   </property>
  </widget>
  <widget class="KScanLineEdit" name="letTPStart">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_TPEnd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TP End :</string>
   </property>
  </widget>
  <widget class="KScanLineEdit" name="letTPEnd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_symbol">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TP symbol :</string>
   </property>
  </widget>
  <widget class="KScanLineEdit" name="letSymbol">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_qam">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Qam type :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_help">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>600</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="KComboEdit" name="cmbQam">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnScan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan Start</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">src/osd/widget_combo_edit.h</header>
  </customwidget>
  <customwidget>
   <class>KScanLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">src/osd/widget_scan_lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
